astroquery/simbad/core.py

```diff
@@ -820,6 +838,51 @@ def query_objectids_async(self, object_name, cache=True):
         response = self._request("POST", self.SIMBAD_URL, data=request_payload,
                                  timeout=self.TIMEOUT, cache=cache)
         return response
+        
+    def query_bibcodelist(self, object_name, verbose=False):
+        """
+        Query Simbad with an object name, and return a table of all
+        bibcodes associated with that object in a `~astropy.table.Table`.
+        (Two columns, 1 row; Main Id with all the bibcodes (separates by";"))
+
+        Parameters
+        ----------
+        object_name : str
+            name of object to be queried
+
+        Returns
+        -------
+        table : `~astropy.table.Table`
+            Query results table
+
+        """
+        result = self.query_bibcodelist_async(object_name)
+        return self._parse_result(result, SimbadBibcodelistResult,
```

Comment:
we need to make sure that `_parse_result` looks at the `SimbadBibcodelistResult` and understands it